src/njoy21.hpp

```diff
@@ -18,10 +18,7 @@
 #include "Log.hpp"
 #include "utility.hpp"
```

Comment:
I would have expected to see something like 

```c++
#include "leapr.hpp"
```
here. If we don't include leapr, then it won't get called. Can you explain why it's not there?

Reply:
I found it

Reply:
Oh heavens. Yes I think I'd added it in and then forgot to come back and formally merge this. I think that all the notes on this PR have already been addressed, should we close/delete it?

Reply:
I'm still doing a little investigating.